test-jw/src/App.js

```diff
@@ -1,10 +1,10 @@
-import { VideoPlayer } from './bundle.esm.js'
-import { WIREWAX_VIDEO, REGULAR_VIDEO, ARTICLE_VIDEO_DETAILS } from './videoConfigs';
+import { CanonicalVideoPlayer } from './bundle.esm.js'
+import { WIREWAX_VIDEO, CANONICAL_VIDEO, ARTICLE_VIDEO_DETAILS } from './videoConfigs';
 
 function App() {
   return (
     <div className="App">
-		<VideoPlayer playlist={REGULAR_VIDEO ?? WIREWAX_VIDEO} />
+		  <CanonicalVideoPlayer currentVideo={CANONICAL_VIDEO ?? WIREWAX_VIDEO} />
```

Comment:
Good to have this test App setup. Might be useful in the future to do some conditional here for all the video players we have, but not right now.